packages/syntax-core/src/SelectList/SelectList.tsx

```diff
@@ -0,0 +1,136 @@
+import React, { ReactElement, ReactNode, useId } from "react";
+import styles from "./SelectList.module.css";
+import classNames from "classnames";
+import Typography from "../Typography/Typography";
+import SelectOption from "./SelectOption";
+
+const iconSize = {
+  sm: 20,
+  md: 24,
+  lg: 24,
+} as const;
+
+const SelectList = ({
+  children,
+  disabled = false,
+  error = false,
+  errorText,
+  helperText,
+  label,
+  onChange,
+  placeholderText,
+  selectedValue = "",
+  size = "md",
+}: {
+  /**
+   * One or more SelectList.Option components.
+   */
+  children: ReactNode;
+  /**
+   * true if the select dropdown is disabled
+   */
+  disabled?: boolean;
+  /**
+   * true if there is an error with the selection
+   */
+  error?: boolean;
+  /**
+   * Text shown below select box if there is an input error.
+   * Should only have a value if error = true
+   */
+  errorText?: string;
+  /**
+   * Text shown below select box
+   */
+  helperText?: string;
+  /**
+   * Text shown above select box
+   */
+  label?: string;
+  /**
+   * The callback to be called when an option is selected
+   */
+  onChange: React.ChangeEventHandler<HTMLSelectElement>;
+  /**
+   * Text showing in select box if no option has been chosen
+   *
+   */
+  placeholderText: string;
+  /**
+   * Value of the currently selected option
+   */
+  selectedValue: string;
+  /**
+   * Size of the select box
+   * * `sm`: 32px
+   * * `md`: 40px
+   * * `lg`: 48px
+   *
+   * @defaultValue "md"
+   */
+  size?: "sm" | "md" | "lg";
+}): ReactElement => {
+  const id = useId();
+
+  return (
+    <div
+      className={classNames(styles.selectContainer, {
+        [styles.opacityOverlay]: disabled,
+      })}
+    >
+      <label htmlFor={id} className={styles.outerTextContainer}>
+        <Typography
+          size={100}
+          color={error ? "destructive-primary" : "gray700"}
+        >
+          {label}
+        </Typography>
+      </label>
+      <div className={styles.selectWrapper}>
+        <select
+          id={id}
+          data-testid="test-select-id"
+          disabled={disabled}
+          className={classNames(styles.selectBox, styles[size], {
+            [styles.unselected]: !selectedValue && !error,
+            [styles.selected]: selectedValue && !error,
+            [styles.selectError]: error,
+          })}
+          onChange={onChange}
+        >
+          {selectedValue === "" && (
+            <option disabled aria-selected value="" selected>
+              {placeholderText}
+            </option>
+          )}
+          {children}
+        </select>
+        <div className={styles.arrowIcon}>
+          <svg
+            focusable="false"
+            aria-hidden="true"
+            viewBox="0 0 24 24"
+            width={iconSize[size]}
+          >
+            <path
+              fill={error ? "#d32a4b" : "#353535"}
```

Comment:
@nathan-ong could we import these colors from `syntax-design-tokens`?

e.g.

```js
import { ColorBaseDestructive700 } from "@cambly/syntax-design-tokens/dist/js/index.js";
```

Reply:
ooh yes, I didn't think to do it this way!